### PR TITLE
feat: add github workflow to publish composedb-cli image

### PR DESCRIPTION
**Pull request by 3benbox**
Publish an image for composedb-cli so that we can use the did tools to convert private keys to dids in keramik.

### CODE REVIEW

File: .github/workflows/docker-cli.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Build Docker Image
+
+on:
+  workflow_dispatch:
+    inputs:
+      COMPOSEDB_CLI_VERSION:
+        description: 'ComposeDB CLI version'
+        default: 'latest'
```
Comment (smrz2001) [inline .github/workflows/docker-cli.yaml:8]: Can we detect the CLI version and use that? I feel like the `latest` tag should apply to all `main` builds and the actual CLI version tag should apply when a new release is created.

File: .github/workflows/docker-cli.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+
+    - name: Build Docker image
+      run: |
+        docker build --build-arg COMPOSEDB_CLI_VERSION=${{ github.event.inputs.COMPOSEDB_CLI_VERSION }} -t ceramicnetwork/composedb-cli:${{ github.event.inputs.COMPOSEDB_CLI_VERSION }} .
+
+    - name: Push Docker image
+      run: |
+        docker push ceramicnetwork/composedb-cli:${{ github.event.inputs.COMPOSEDB_CLI_VERSION }}
```
[the remark this answers is not in the document]
Comment (3benbox) [reply .github/workflows/docker-cli.yaml:28]: Added!

File: .github/workflows/docker-cli.yaml (hunk 1 above)
Comment (3benbox) [reply .github/workflows/docker-cli.yaml]: Updated to take a branch and version from that branch.

Unsure how the releases are created.

**Review by smrz2001 — APPROVED** [review]
Can figure out how to only tag the release after a release commit later, but otherwise LGTM.

File: .github/workflows/docker-cli.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          push: true
+          build-args: |
+            VERSION=${{ steps.package-version.outputs.version }}
+          tags: ${{ env.DOCKER_IMAGE }}:${{ steps.package-version.outputs.version }}
```
Comment (smrz2001) [inline .github/workflows/docker-cli.yaml:44]: Can you add the `latest` tag here as well since the image is tracking `main`?

Comment (3benbox) [reply .github/workflows/docker-cli.yaml:44]: Added!

**Review by smrz2001 — APPROVED** [review]
🚀

**Pull request merged**